feature-libs/product/configuration/variant/src/components/variant-components.module.ts

```diff
@@ -0,0 +1,4 @@
+import { NgModule } from '@angular/core';
+
+@NgModule({})
+export class VariantComponentsModule {}
```

Comment:
Here I propose VariantConfiguratorComponentsModule (otherwise confusion with product variants might be possible)